src/routes/pages/en/guides/start-posting-notes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Start Posting Notes
+description: Now that you're on nostr, start posting notes (and other stuff!)
+---
+
+
+## [§](#keys-events-relays) Keys, Events, and Relays – Oh My!
+
+You've downloaded a nostr client, created your cryptographic keypair (ie your login credentials), and connected to a few relays. Now, you’re ready to start posting notes.
+
+In this guide, you'll learn the basics of posting on nostr: content, media, hashtags, community engagement, and value-for-value :zap:
+
+## [§](#introductory-post) Introductory Post
+
+There are various ways to connect with the communities of nostr, but it never hurts to begin by introducing yourself!
+
+With no character limit, you can say more than just "hello world". Let people know how you found nostr and why you're here. And don't forget to add a couple of hashtags, such as #introductions, to help people see your post. 
+
+In nostr's early days, it could be difficult to search for specific words or people, but that is quickly changing with some nostr clients introducing more robust search functionality. One of the best ways to discover new content is by using and following hashtags.
+
+## [§](#utilizing-hashtags) Utilizing hashtags
+
+Hashtags are supported in most nostr clients. It's great way to find content that interests you, as well as showing your posts to other users that you haven't connected with before. 
+
+At the time of this writing, the hashtag #grownostr is a popular catch-all, used for a wide range of topics. (Since nostr was built by bitcoiners, there is still a lot of bitcoin-related discussion, so we use #grownostr to identify various other topics). 
+
+A few other popular hashtags include #nature, #artstr, #photography, #foodstr, #dogstr, and #AiArt, for example. Search or post with #introductions and #plebchain to find new friends.
+
+Ultimaetly, you can Use any hashtags that help you find interesting content. If you don't see much, try adding "-str" to the end. For example, if you're looking for cute cat photos, search for #catstr :cat2:
+
+
+## [§](#tagging-other-users) Tagging Other Users
+
+Tagging other users is still a work in progress (as of this writing in early 2024). On most clients, if you're already following someone, then using @ followed by their display name will work. Of course, if you use @ followed by their npub, that's a surefire way to tag your friend (but of course that assumes you have already saved their npub to your clipboard).
```

Comment:
I think that it'd be good to mention briefly that mentions can also be done by using the `nostr:note1...` or nostr:npub...` syntax. You don't need to go into detail on how to get that data but that way people will recognize it if they see it.